### PR TITLE
[NRLF 296] onboarding changes

### PR DESCRIPTION
**Pull request by Rohoolio**
Adjustments to the readme

### CODE REVIEW

**Review by nomad3k — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -58,9 +91,29 @@ poetry shell
...
+
+```shell
+poetry shell
+source nrlf.sh
+nrlf make build
+nrlf aws reset-creds
+nrlf aws login mgmt <mfa>
+nrlf truststore pull-server dev
+nrlf truststore pull-client dev
+nrlf terraform plan <yourname>-test
```
Comment (andrewrae5-nhs) [inline README.md:108]: Make sure terraform is installed! Add onto the install reqs 😅

Comment (andrewrae5-nhs) [reply README.md:108]: the commends i used are in the terraform docs [here](https://developer.hashicorp.com/terraform/downloads), but the main commands are:
```
wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor | sudo tee /usr/share/keyrings/hashicorp-archive-keyring.gpg
 echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
 sudo apt update && sudo apt install terraform
```

**Review by andrewrae5-nhs — CHANGES_REQUESTED** [review]
This is good - there are a few comments that need to be addressed though. 

also, it might be worthwhile mentioning the [confluence article ](https://nhsd-confluence.digital.nhs.uk/display/CLP/NRLF+-+Developer+Onboarding)at the top, as there are more details in that. 

Actually, I would say that you could probably just get rid of the rest after the quick install and re-direct them to the confluence article, as people will probably not want to keep reading once it's up and running? (Maybe just add a line about getting the AWS credentials sorted)

File: README.md [hunk 2]
```diff
@@ -48,6 +42,45 @@ Swagger generation requirements.
...
+### 2. Linux set up
+
+For those on a linux/WSL setup these are some helpful instructions:
+
+Poetry:
+
+```shell
+curl -sSL https://install.python-poetry.org | python3 - --version 1.1.15 (only use --version to specify)
+nano ~/.bashrc
+add to bashrc - export PATH="/yourHomeDirectory/.local/bin:$PATH" spineVM home dir is "/home/spineii-user/"
```
[the remark this answers is not in the document]
Comment (andrewrae5-nhs) [reply README.md:54]: quick update - check out my comment on line 70 - you can just replicate that styling i put in the comment

**Pull request merged**